cmd/src/batch_common.go

```diff
@@ -9,11 +9,13 @@ import (
 	"os/exec"
```

Comment:
There's a file called `batch_remote` in this directory that uploads the spec file for SSBC execution, we should add a gate there that the `validateMountPaths` method rejects if there is a mount entry present. The API should ideally reject this anyways (to be implemented), but a nice actionable error message can't hurt, I think.